mmv1/products/compute/ForwardingRule.yaml

```
@@ -92,6 +92,15 @@ examples:
       network_name: 'website-net'
     ignore_read_extra:
       - 'port_range'
+  - name: 'forwarding_rule_externallb_byoipv6'
+    primary_resource_id: 'default'
+    min_version: 'beta'
+    vars:
+      forwarding_rule_name: 'test-byoipv6-forwarding-rule'
+      backend_name: 'website-backend'
+      network_name: 'website-net'
```

You can use `test_vars_overrides` to set a variable's value to the result of a function call. I'm not exactly sure what the range of valid values for the IP address would be but it might look something like this:

```suggestion
    vars:
      forwarding_rule_name: 'test-byoipv6-forwarding-rule'
      backend_name: 'website-backend'
      network_name: 'website-net'
      ip_address: '2600:1901:4457:1::/96`
    test_vars_overrides:
      ip_address: 'fmt.Sprintf("2600:%d:%d:%d:%d::/96', rand.Intn(9999) + 1, rand.Intn(9999) + 1, rand.Intn(9999) + 1, rand.Intn(9999) + 1)' 
```